### ISSUE TITLE
How can I attack a image which under normalization?

### ISSUE THREAD

**Issue by CHENBIN99** [Harry24k/adversarial-attacks-pytorch#68]
How can I add perturbation on a image which process by normalization from dataloader. Many model was trained under norm, so I must keep this processing. I have saw the comments in the code:
> images: :math:`(N, C, H, W)` where `N = number of batches`, `C = number of channels`,        `H = height` and `W = width`. It must have a range [0, 1].
It means that it is not supported the norm yet?
